README.md

```diff
@@ -4,6 +4,23 @@
 All endpoints are prefixed with `/api`
 
 ### /requests
+
+#### GET /requests
+The get request will return all the requests in the database.
+The format is:
+```
+{
+  requests: [
+    {
+      id: 1,
+      title: "Title",
+      description: "Lots of text",
+      requester: "requester@mail.com"
```

Comment:
That is not what your serializer returns

Reply:
In the actual response the keys are wrapped as "key".
Is there a functional difference?